roslibrust/examples/native_ros1.rs

```diff
@@ -0,0 +1,31 @@
+/**
+ * Testing ground for hitting on rosmaster directly
+ */
+
+#[cfg(feature = "ros1")]
```

Comment:
You can specify examples with required features: https://github.com/stm32-rs/stm32f1xx-hal/blob/master/Cargo.toml#L84